src/components/Textfields/NormalTextField/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { InputAdornment } from "@material-ui/core";
+import TextFieldBase, { TextfieldBaseProps } from "src/components/Textfields/TextFieldBase";
+
+export interface NormalTextFieldProps extends TextfieldBaseProps {
+    icon?: React.ReactNode;
+}
+
+const NormalTextField = (props: NormalTextFieldProps) => {
```

Comment:
I think we should not name it "normal" 
Please think of another name using this component's feature/purpose